PyInstaller/lib/modulegraph/_compat.py

```diff
@@ -3,7 +3,27 @@
 if sys.version_info[0] == 2:
     def Bchr(value):
         return chr(value)
+    from StringIO import StringIO as BytesIO
+    from StringIO import StringIO
+    from urllib import pathname2url
+    _cOrd = ord
 
+    # File open mode for reading (univeral newlines)
+    _READ_MODE = "rU"
+
+    PY2 = True
 else:
+    from urllib.request import pathname2url
+    from io import BytesIO, StringIO
+    _cOrd = int
+    _READ_MODE = "r"
+
     def Bchr(value):
         return value
+
+    PY2 = False
+
+if sys.version_info < (3, 4):
+    from xdis.std import get_instructions
```

Comment:
I don't care much about Py 3.3, too.

> The reson for this is …

I know, and this is good. But the compatibility code for Python 2 already exists in PyInstaller. So there is no need to depend on an unmaintained (dis3) or huge (xdis) module.